server/src/main/java/org/candlepin/auth/permissions/JobStatusPermission.java

```diff
@@ -32,13 +32,13 @@
  * created/initiated.
  *
  */
```

Comment:
Please update the javadoc to reflect that this is a general JobStatus permission and give a little info about what it allows access to.